lib/clearance/back_door.rb

```diff
@@ -31,8 +31,9 @@ module Clearance
   #   visit new_feedback_path(as: user)
   class BackDoor
     def initialize(app, &block)
-      unless ENV["RAILS_ENV"] == "test"
-        raise "Can't use backdoor outside test environment"
+      @allowed_envs = Clearance.configuration.allowed_envs || ["test", "ci", "development"]
```

Comment:
If we set these defaults in the config itself, we probably don't need to set here?

Reply:
Agreed, resolved.